app/views/application/createEvent.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Choose event format</h1>
+
+<%= link_to '<button type="button">New league</button>'.html_safe, new_event_path %>
```

Comment:
This should also definitely not be in dev untill new_event_paths is defined. So we should probably merge #19 with this before merging into dev. Otherwise this link will not be defined.